Formula/llvm.rb

```diff
@@ -214,6 +217,30 @@ def install
         # Make sure CMake doesn't try to pass C++-only flags to C compiler.
         extra_args << "-DCMAKE_C_COMPILER=#{ENV.cc}"
         extra_args << "-DCMAKE_CXX_COMPILER=#{ENV.cxx}"
+
+        # Link to libstdc++ for brewed GCC rather than the host GCC which is too old.
+        linux_library_paths = [
+          Formula["gcc"].opt_lib/"gcc"/gcc_version, # libstdc++
+          Formula["glibc"].opt_lib,
+        ]
+
+        # Use libstdc++ headers for brewed GCC rather than host GCC which is too old.
+        # We also need to make sure we can find headers for other formulae on Linux.
+        linux_include_paths = [
+          Formula["gcc"].opt_include/"c++"/gcc_version,
+          Formula["gcc"].opt_include/"c++"/gcc_version/"x86_64-pc-linux-gnu",
+          HOMEBREW_PREFIX/"include",
+        ]
```

Comment:
```suggestion

        gcc = Formula["gcc"]
        # Link to libstdc++ for brewed GCC rather than the host GCC which is too old.
        linux_library_paths = [
          gcc.opt_lib/"gcc"/gcc.version.major, # libstdc++
          Formula["glibc"].opt_lib,
        ]

        # Use libstdc++ headers for brewed GCC rather than host GCC which is too old.
        # We also need to make sure we can find headers for other formulae on Linux.
        linux_include_paths = [
          gcc.opt_include/"c++"/gcc.version.major,
          gcc.opt_include/"c++"/gcc.version.major/"x86_64-pc-linux-gnu",
          HOMEBREW_PREFIX/"include",
        ]
```